### ISSUE TITLE
With Split editor, Problems in right-hand pane reopen file in left-hand pane

### ISSUE THREAD

**Issue by adamesque** [Microsoft/vscode#49227]
In Split view, when editing in the right-hand buffer, if I save the file I'm working in and there's a new Problem (contributed from, say, the ESlint plugin), the Problems/output panel opens, and the file with the problem _is opened again_ in the left-hand buffer and focus is moved there!

This is insanely frustrating.

I've tried setting `problems.autoReveal` to `false`, with no effect.

(Apologies — I'm not sure I'm using the correct terms for any of these features!)

- VSCode Version: 1.23.0
- OS Version: Mac OS X 10.13.2

Steps to Reproduce:

1. Enable an extension that contributes problems on save (like eslint)
2. Split the editor left/right
3. Add some broken code in the secondary pane
4. Save the file

Results:
- Problems/output pane opens on the bottom
- File from right-hand split is reopened in left-hand pane
- Focus is moved to left-hand pane

Does this issue occur when all extensions are disabled?
No, but I'm not sure how to trigger a Problem without extensions enabled.

**Comment by sandy081** [comment]
@adamesque Do you have any extensions other than `eslint` ?

**Comment by adamesque** [comment]
After some more careful observation, I think there's a chance this is caused by the `Prettier Now` extension, which has two settings that default to `true` that may have been causing this:
- `prettier.openOutput`: Automatically opens Prettier output when an error is detected
- `prettier.autoScroll`: Scroll automatically to line where error has been detected

I've disabled these settings and will see if that resolves the issue. Closing this issue for now; will reopen if that isn't the root cause. Thanks for asking — your probing question is what prompted me to look more closely.
